sdk/digitaltwins/Azure.DigitalTwins.Core/perf/Azure.DigitalTwins.Core.Perf/Infrastructure/PerfTestEnvironment.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.TestFramework;
+
+namespace Azure.DigitalTwins.Core.Perf
+{
+    /// <summary>
+    /// Represents the ambient environment in which the test suite is being run, offering access to information such as environment variables.
+    /// </summary>
+    internal sealed class PerfTestEnvironment : TestEnvironment
+    {
+        /// <summary>
+        /// The shared instance of the <see cref="PerfTestEnvironment"/> to be used during test runs.
+        /// </summary>
+        public static PerfTestEnvironment Instance { get; } = new PerfTestEnvironment();
+
+        /// <summary>
+        ///   The Digital Twins instance endpoint to run the tests against.
+        /// </summary>
+        public string DigitalTwinsUrl => GetVariable("DIGITALTWINS_URL");
```

Comment:
These environment variables are set during the live test pipeline resource deployment. I assume we have to use the same ones for the perf tests (since it was mentioned that we write them just as we would write Live tests)

Reply:
It turns out that there is currently no automation in perf test framework and the perf team will run the tests manually (with my help of course)